NEWS.rst

```diff
@@ -91,6 +91,9 @@ the representation of origin-spanning locations. See issue #4611.
 
 Updated ``Bio.Restriction`` to the April 2024 release of REBASE.
 
+A bug in ``bgzf`` was resolved, restoring the ability to pass a file handle
+directly to ``BgzfWriter``.
```

Comment:
Based on historical precedent, I wouldn't normally have added a news entry for a relatively minor bug fix, but it can stay unless the release notes get too long.

(Plenty of other projects go to the other extreme of reporting everything which would be excessive here)

Reply:
Thanks, Peter! By all means please feel free to purge this entry later -- my feelings won't be hurt if this super trivial fix doesn't make the release notes! 😄